cdap-ui/app/cdap/components/AbstractWidget/AbstractMultiRowWidget/index.tsx

```diff
@@ -59,8 +59,19 @@ export default class AbstractMultiRowWidget<
     this.init(nextProps);
   }
 
-  private init = (props) => {
+  public deconstructValues = (props) => {
```

Comment:
does this method need to be public to be overwritten?

Reply:
I get the following error if I make it private: 
`Types have separate declarations of a private property 'deconstructValues'`